webui/react/src/components/TasksTable.tsx

```diff
@@ -0,0 +1,100 @@
+import { Table } from 'antd';
+import { ColumnsType } from 'antd/lib/table';
+import React from 'react';
+import TimeAgo from 'timeago-react';
+
+import Avatar from 'components/Avatar';
+import Badge from 'components/Badge';
+import linkCss from 'components/Link.module.scss';
+import { CommonProps, Task } from 'types';
+import { alphanumericSorter, stateSorter, stringTimeSorter } from 'utils/data';
+import { canBeOpened } from 'utils/task';
+
+import { BadgeType } from './Badge';
```

Comment:
thought: maybe just converge to `./` path instead of `components` when referencing a component from another component?